atc/api/jobs_test.go

```diff
@@ -1404,6 +1417,19 @@ var _ = Describe("Jobs API", func() {
 						"team_name": "some-team",
 						"start_time": 101,
 						"end_time": 200
+					},
+					{
+						"id": 3,
+						"name": "1.1",
+						"job_name": "some-job",
+						"status": "succeeded",
+						"api_url": "/api/v1/builds/3",
+						"pipeline_name": "some-pipeline",
+						"team_name": "some-team",
+						"start_time": 102,
+						"end_time": 300,
+						"rerun_of": "1",
```

Comment:
I don't think it's super obvious what this value is. What about something like this?:

```json
{
  "rerun_of": {
    "id": 1234,
    "name": "1",
    "api_url": "/api/v1/builds/2"
  }
}
```